lib/ruby_lsp/tapioca/server_addon.rb

```diff
@@ -71,6 +77,18 @@ def dsl(constants, *args)
 
         ::Tapioca::Cli.start(arguments)
       end
+
+      def gem(added_or_modified_gems, removed_gems)
```

Comment:
The `removed_gems` parameter is unused. Tests are probably passing because we're invoking Tapioca every time, but removing orphaned RBIs with the current approach in main is much faster.